extensions/wikia/BannerNotifications/js/BannerNotifications.js

```diff
@@ -14,25 +14,57 @@
 (function (window, $, mustache) {
 	'use strict';
 
-	var defaultTimeout = 10000,
+	var templates = {
+			bannerNotification: '<div class="wds-banner-notification {{typeClassName}}">' +
+			'<div class="wds-banner-notification__icon">' +
+			'{{{icon}}}</div>' +
+			'<span class="wds-banner-notification__text">{{{content}}}</span>' +
+			// DS icon: wds-icons-cross-tiny
+			'<svg class="wds-icon wds-icon-tiny wds-banner-notification__close" width="12" height="12" viewBox="0 0 12 12" xmlns="http://www.w3.org/2000/svg">' +
```

Comment:
can you add here a TODO comment with the ticket number for supporting ds icons in JS?